src/camvidlog2/cli.py

```
@@ -95,37 +99,41 @@ def query(
-        query_max.sort_values(
-            by=j,
-            ascending=False,
-            inplace=True,
-        )  # type: ignore
-        # this is a typing bug - it is legit to have non-string column names
-
+    # output for an average of all distances
+    distances["avg"] = distances.mean(axis=1)
+    for result in calculate_results(distances["avg"], num, roll):
+        print(
+            f"avg {result.rank:3d} {result.frame_no:4d} {result.score:.3f} {result.filename}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)